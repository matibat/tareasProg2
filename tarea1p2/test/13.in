leer_info (50,abc de)
copiar_info
imprimir_numero
imprimir_frase
imprimir_info
cambiar_numero 51
copiar_info
imprimir_info
cambiar_frase efg hi
copiar_info
imprimir_info
combinar_info (50,aaa) (-50, bb)
copiar_info
imprimir_numero
imprimir_frase
imprimir_info
son_iguales (50,aaa) (-50, bb)
Fin
