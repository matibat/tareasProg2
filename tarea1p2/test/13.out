1>El nuevo info es (50,abc de).
2>La copia es (50,abc de).
3>El dato numérico es 50.
4>El dato de texto es abc de.
5>El info es (50,abc de).
6>El nuevo número es 51.
7>La copia es (51,abc de).
8>El info es (51,abc de).
9>la nueva frase es efg hi.
10>La copia es (51,efg hi).
11>El info es (51,efg hi).
12>El info combinado es (0,aaa bb).
13>La copia es (0,aaa bb).
14>El dato numérico es 0.
15>El dato de texto es aaa bb.
16>El info es (0,aaa bb).
17>No son iguales.
18>Fin.
